ruby_hack_assembler.rb

```diff
@@ -0,0 +1,6 @@
+# Run `ruby ruby_hack_assembler.rb examples/add.asm`
```

Comment:
We don’t have any tests for `ruby_hack_assembler.rb`. How would we write an acceptance test to check that it does what it’s supposed to, i.e. produces the expected output when run from the shell with a given input? [Here’s one possible starting point.](https://blog.stigok.com/2020/08/29/jekyll-ruby-python-plugin-subprocess-open3-exec-command.html)

Reply:
Done!